config.json

```diff
@@ -660,16 +660,16 @@
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-annotations",
-        "version": "16.0.0",
-        "nugetVersion": "116.0.0.7",
+        "version": "16.1.0",
+        "nugetVersion": "116.1.0",
         "nugetId": "Xamarin.Firebase.Annotations",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-appcheck-interop",
-        "version": "16.0.0-beta01",
-        "nugetVersion": "116.0.0.7-beta01",
+        "version": "16.0.0-beta04",
+        "nugetVersion": "116.0.0.4-beta04",
```

Comment:
I think this is a version downgrade:
`116.0.0.4-beta04` is considered "before" `116.0.0.7-beta01`.

This should probably be `116.0.0.7-beta04`.

Reply:
fixed